src/ws-dropdown-item.js

```diff
@@ -0,0 +1,126 @@
+import {React, Component} from './imports';
+import {WSDropdownMenu} from './ws-dropdown-menu';
+
+export class WSDropdownItem extends Component {
+
+  /**
+   * @type {Object}
+   */
+  static defaultProps = {
+    item: null,
+    icon: null,
+    multiple: false,
+    isParent: false,
+    handle: () => {}
+  };
+
+  /**
+   * @type {Object}
+   */
+  static propTypes = {
+    item: React.PropTypes.object,
+    icon: React.PropTypes.string,
+    multiple: React.PropTypes.bool,
+    isParent: React.PropTypes.bool,
+    handle: React.PropTypes.func
+  };
+
+  /**
+   * @type {Object}
+   */
+  static contextTypes = {
+    multiple: React.PropTypes.bool
+  };
+
+  /**
+   * @param props {Object}
+   * @param context {Object}
+   * @constructor
+   */
+  constructor(props, context) {
+    super(props, context);
+    this.state = props.item;
+    this.menu = null;
+  }
+
+  /**
+   * Important to update the state when the items in menu changed.
+   * When in the multi select a dropdown item is selected and stored (submitted) it will be filtered out
+   * of the item list and shown in a separate value list. React doesn't created new components but reuses them.
+   * This leads to property updates and without this function a wrong item will be rendered.
+   * @param props {Object}
+   * @void
+   */
+  componentWillReceiveProps(props) {
+    this.state = props.item;
```

Comment:
I'd like to use the state since selected and stored can change and I don't want to build a functionality to propagate it. Using the item as the state the parent knows automatically whats going on because it's just a reference.